pymagicc/__init__.py

```diff
@@ -1,6 +1,11 @@
 # Copyright (c) 2017 pymagicc authors:
-#   Robert Gieseke <robert.gieseke@pik-potsdam.de>
-# Free software under GNU Affero General Public License v3, see LICENSE
+#   - Robert Gieseke <robert.gieseke@pik-potsdam.de>
+#   - Zeb Nicholls <zebedee.nicholls@climate-energy-college.org>
+#   - Jared Lewis <jared.lewis@climate-energy-college.org>
+#   - Sven Willner <sven.willner@pik-potsdam.de>
+#   - Matthias Mengel <matthias.mengel@pik-potsdam.de>
+#
+# Free software. For license details, see LICENSE.
```

Comment:
```suggestion
# For license details, see LICENSE.
```
Not "free" software anymore ... :smile_cat: Still open of course ...

Reply:
I've removed these comments anyway because pandas doesn't have anything like this anyway. However, how is it not free anymore?

Reply:
See above, i was confused ...

Reply:
I think the politics of "free software" are especially tricky in this repo as the binary is clearly not "free software". It's written that they have different licenses but it's still confusing ...

Reply:
Ok all good